couchbase/json-queries.yml

```diff
@@ -0,0 +1,151 @@
+type: json
+data:
+  url: /admin/vitals
+  server: ${COUCHBASE_QUERY_HOSTPORT}
+  basicHttpAuthUsername: ${COUCHBASE_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${COUCHBASE_BASICAUTH_PASSWORD}
+
+observation:
+  - name: queries
+    metricNamespace: couchbase
+    path: $.
+
+    metric:
+      - name: cores
+        source: cores
+        type: long_gauge
+        label: cores
+        description: Cores
+
+      - name: cpu.sys.percent
+        source: cpu.sys.percent
+        type: long_gauge
+        label: system cpu
+        unit: "%"
+        description: Percentage of CPU used by the system
+
+      - name: cpu.user.percent
+        source: cpu.user.percent
+        type: long_gauge
+        label: user cpu
+        unit: "%"
+        description: Percentage of CPU used by the user
+
+      - name: gc.num
+        source: gc.num
+        type: long_gauge
+        label: gc num
+        description: Number of objects garbage collected
+
+      - name: gc.pause.percent
+        source: gc.pause.percent
+        type: long_gauge
+        label: gc pause percent
+        unit: "%"
+        description: Garbage collection pause percentage
+
+      - name: gc.pause.time
+        source: gc.pause.time
+        type: long_gauge
+        label: gc pause time
+        unit: seconds
+        description: Garbage collection pause time
+
+      - name: memory.system
+        source: memory.system
+        type: long_gauge
+        label: system memory
+        unit: bytes
+        description: Memory used by the system
+
+      - name: memory.total
+        source: memory.total
+        type: long_gauge
+        label: total memory
+        unit: bytes
+        description: Memory used by Couchbase over the total period of time
+
+      - name: memory.usage
+        source: memory.usage
+        type: long_gauge
+        label: usage memory
+        unit: bytes
+        description: Memory currently used by Couchbase
+
+      - name: request.active.count
+        source: request.active.count
+        type: long_gauge
+        label: active requests
+        description: Number of active requests
+
+      - name: request.completed.count
+        source: request.completed.count
+        type: long_gauge
+        label: requests completed
+        description: Number of requests completed
+
+      - name: request.per.sec.15min
+        source: request.per.sec.15min
+        type: long_gauge
+        label: requests/sec in 15 min
+        description: Requests per second in the last 15 minutes
```

Comment:
these rate metrics may be useful for us. right @bsmid ?

Reply:
Typically they are useless since we have no idea about exact 15 min range and we can't convert them to something with max 1min granularity anyway. We could collect and display them only if nothing else is exposed or there is no way to recalculate them into real counters.

Reply:
Did you find better replacements? If not, keep them.